test/it/uk/gov/hmrc/exports/connector/CustomsDataStoreConnectorSpec.scala

```diff
@@ -18,37 +18,72 @@ package uk.gov.hmrc.exports.connector
 
 import java.time.ZonedDateTime
 
-import scala.concurrent.ExecutionContext.Implicits.global
-
 import play.api.libs.json.Json
-import play.api.test.Helpers.OK
+import play.api.test.Helpers._
 import testdata.ExportsTestData
-import uk.gov.hmrc.exports.config.AppConfig
 import uk.gov.hmrc.exports.base.IntegrationTestSpec
+import uk.gov.hmrc.exports.config.AppConfig
 import uk.gov.hmrc.exports.connectors.CustomsDataStoreConnector
 import uk.gov.hmrc.exports.models.VerifiedEmailAddress
-import uk.gov.hmrc.http.HttpClient
 
 class CustomsDataStoreConnectorSpec extends IntegrationTestSpec {
 
-  implicit val appConfig: AppConfig = inject[AppConfig]
+  implicit private val appConfig: AppConfig = inject[AppConfig]
+  private val connector = inject[CustomsDataStoreConnector]
```

Comment:
Not sure, but I think you will have two different `AppConfig` instances here.

Reply:
Since it is injected, it depends on how `AppConfig` is defined. But I had a look and it has `@Singleton` annotation, so there should be just a sinigle instance.